### ISSUE TITLE
Minimal repeated section: missing "Remove" icon title on hover

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#5009]
For the minimal repeated grid, the `title` attribute shows, but not for the minimal repeated section, which has an `aria-label`.

**Comment by ebruchez** [comment]
Also, the `aria-label` was pointing to the wrong resource (grid instead of section).
